### ISSUE TITLE
PostgreSQL Row Security Policies (RLS)

### ISSUE THREAD

**Issue by ErunamoJAZZ** [dbeaver/dbeaver#5499]
Hello, first, thanks for this awesome software, you guys are doing a great work making something really usable to manage databases!

There is a feature that allow to manage rules in postgres ( https://github.com/dbeaver/dbeaver/issues/2284 ).

I think it could be good to know something similar for Policies: https://www.postgresql.org/docs/current/ddl-rowsecurity.html

#### Steps to reproduce, if exist:
```sql
CREATE TABLE public."user" (
	id serial NOT NULL,
	"name" text NULL
);

CREATE POLICY user_read_all_policy ON public.user
  FOR SELECT
  USING (true);

CREATE POLICY user_insert_any_policy ON public.user
  FOR INSERT
  WITH CHECK(true);

ALTER TABLE public.user ENABLE ROW LEVEL SECURITY;
```

Actually, the way to see what policies have a table, we can use:
```sql
select * from pg_policies;
```
![imagen](https://user-images.githubusercontent.com/421398/54392587-11c94880-4676-11e9-9389-f3e6b7943bf6.png)

----------------
I hope you can consider this for future releases.

**Comment by pyrocks** [comment]
Would be nice to have a `Policies` tab on the list under a Table's `Properties` tab, similar to Indexes, Triggers, Constraints, Partitions, Rules etc. I use policies on almost all tables.

**Comment by mortifia** [comment]
hello, it would be really good to see the rls with the tables

**Comment by caburet** [comment]
Its gonna be nice to have it!

**Comment by megafetis** [comment]
Its gonna be nice to have it!!

**Comment by ozkankirik** [comment]
+1

**Comment by ShadelessFox** [comment]
Implemented and will be available in DBeaver 22.1.3:

![image](https://user-images.githubusercontent.com/35821147/178475244-7f257b1d-1042-4c10-a60f-d1081dc114cb.png)
